### DISCUSSION TITLE
There is a problem starting webui."Running on local URL" is gone!

### DISCUSSION THREAD

**Discussion by Dany19212050** [oobabooga/text-generation-webui#1756] [Q&A]
Follow the steps on the Git hub to .zip oobabooga-windows, download and unzip and install, and also download and install the model, why does not appear Running on local URL: https//127.0.0.1:7860...when opening?'Running on local URL: https//127.0.0.1:7860"is gone!!To solve this problem, I reinstalled it again.But it still shows this.
<img width="1275" alt="屏幕截图 2023-05-03 185909" src="https://user-images.githubusercontent.com/132433940/235887061-b981a570-c43f-4c6e-b63e-82263c5848ba.png">

This error also occurred on the second installation.Could this be the cause of this error?
<img width="1278" alt="屏幕截图 2023-05-03 173737" src="https://user-images.githubusercontent.com/132433940/235869293-ba43ed97-c0e6-40c7-b42e-03ca275cf592.png">
<img width="1274" alt="屏幕截图 2023-05-03 173756" src="https://user-images.githubusercontent.com/132433940/235869296-61eba059-f967-41e9-9302-ae5c99a4eb89.png">

**Comment by VvvVVv-v** [comment]
I got the same problem, did you manage to sort it out?
